docs/release/handbook.md

```diff
@@ -0,0 +1,288 @@
+
+**Responsibilities:**
+
+* Keeping track of the provided component versions and updating the [versions table](https://github.com/kubeflow/manifests#kubeflow-components-versions)
+* Creating PRs to update manifests repo from WG provided git revisions
+* Attending the Release Team meetings
+* Providing feedback on enhancing the current Release Handbook
+
+### Product Manager
+They will be in charge of driving non-code deliverables, like the post-release blog post and social media announcements. The Product Manager is also responsible for tracking important features and bug fixes that should be highlighted in the release.
+
+**Prerequisites:** Experience in developing blog posts. Contributions to an existing Open Source project is a huge plus.
+
+**Time Commitments:** This role will require synchronous communication, since they need to be attending the Release Team meetings. Most of the effort is expected to start once the Feature Freeze phase completes.
+
+**Responsibilities:**
+
+* Driving the blog post effort
+* Keeping notes from the Release Team meetings
+* Making sure the processes are being followed
+* Handling communication for social media content publication
+
+### Docs Lead
+The Docs Lead is responsible for working with the Release Team to coordinate documentation updates for the next Kubeflow release.
+
+**Prerequisites:** Contributions to the [website repo](https://github.com/kubeflow/website)
+
+**Time Commitments:** This role will require some synchronous communication. The bulk of their work is expected to be in the Documentation phase of the release, but they will also be involved throughout the release by keeping track of issues that might require an update to the docs.
+
+**Responsibilities:**
+* Identifying and tracking new issues that require update to the docs
+* Working with contributors to modify existing docs to accurately represent any upcoming changes
+* Reviewing documentation PRs to ensure quality following the website [Style Guide](https://www.kubeflow.org/docs/about/style-guide/)
+* Migrating the old website [version] documentation and updating it with the new release
+
+### Working Group Liaison
+Person responsible for liaising between the release team and the WG. The will
+be the main contact point for the Release Team to figure out the progress of
+the WG as well as required component versions, throughout the release.
+
+**Prerequisites:** Contributor in the Working Group and acknowledged from the tech leads of that Working Group
+
+**Time Commitments:** This role can be completely asynchronous. Most of their work is expected to be in the Feature Freeze and Documentation phases.
+
+**Responsibilities:**
+- Providing a roadmap and dependency updates to the Community early in the release cycle
+- Communicating with Release Team during the Feature Freeze about the progress of bug fixes
+
+### Distribution Representative
+
+**Prerequisites:** Work at the organization that owns the distribution
```

Comment:
We are still lacking a clear definition of what a distribution is and what requirements a distribution must fulfill to be an "official" / "certified" distribution and listed on the website.

Reply:
Stay tuned, we'll present sth on this topic separately.

Reply:
Thanks for the heads up @Bobgy, I'll move on and resolve this comment.

Once we have a more clear picture and definition of what a distribution is we can create a PR and amend this section